Program/StringFormatting.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text;
+using HotelBooking;
+
+namespace Program
+{
+    public static class StringFormatting
```

Comment:
Should have:

Class name issues:
- In first place we should consider class name to be noun not verb (not always true, but usually class comes from nouns)
- StringFormating sounds very general and your class to very specific thing.  So I would suggest to name this like:
  - ReservationResultTextFormatter
  - ReservationResultConsolePresenter

Reply:
Applied